spec/tasks/job_board/digest_spec.rb

```diff
@@ -5,18 +5,35 @@
 Rails.application.load_tasks
 
 RSpec.describe 'job_board:digest' do
-  after(:each) do
-    Rake::Task['job_board:digest'].reenable
+  after(:each) { Rake::Task['job_board:digest'].reenable }
+
+  context 'when it is Monday' do
+    # April 11, 2022 was a Monday
+    before { Timecop.travel(Time.new(2022, 4, 11)) }
```

Comment:
love using timecop here!! out of curiosity I did some googling and found some cool ActiveSupport methods, so we can do something like `Date.today.last_week(:monday)` and avoid the magic dates in this file :)

https://api.rubyonrails.org/classes/DateAndTime/Calculations.html#method-i-last_week

Reply:
Okay Stephanie... this is an awesome suggestion, but I feel like I am losing my mind. Can you pull down this branch and run the tests on your machine? When I run them locally, they fail because when I write `Timecop.travel(Date.today.last_week(:monday))`, it changes the date to last Sunday? But it passes on CI? Is something wrong with my computer?